docs/src/pages/guides/data-fetching.md

```diff
@@ -7,7 +7,7 @@ Astro components and pages can fetch remote data to help generate your pages. As
 
 ## `fetch()`
 
-Astro pages have access to the global `fetch()` function in their setup script. `fetch()` is a native JavaScript API ([MDN](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)) that lets you make HTTP requests for things like APIs and resources.
+Astro pages have access to the global [`fetch()`][mdn-fetch] function in their setup script. [`fetch()`][mdn-fetch] is a native JavaScript API that lets you make HTTP requests for things like APIs and resources.
```

Comment:
this was a stylistic choice. not opposed to the new change, but then only link the first reference.

Reply:
I'm not 100% sure about which is better, there's an a11y rule that says "links with the same name should do the same thing" so having multiple MDN links that link to different places isn't amazing... maybe 
```html
<a href="https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch" aria-label="MDN - Using Fetch">MDN</a>
```
would work to make that better? Since the idea is that links should contain all the info required to understand them from within them.

Reply:
It looks like there's a way to add a `title` in markdown 🤔 I don't remember whether that's accessible to screen readers though...
https://www.markdownguide.org/basic-syntax#adding-titles

Reply:
From https://www.deque.com/blog/text-links-practices-screen-readers/ and https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA8.html it appears that `aria-label` is the best practice.

Reply:
Or maybe good ole' `span.sr-only` can do the job 😆

Reply:
<img width="700" alt="image" src="https://user-images.githubusercontent.com/10626596/126694759-b6d9c331-30f4-4c4e-86a9-6de865fc23fd.png">
In Firefox a11y tools xD

Reply:
That works! I believe it's pretty standard practice to only link to the first reference to something. Kind of like:

<img width="756" alt="Screen Shot 2021-07-22 at 12 51 10 PM" src="https://user-images.githubusercontent.com/622227/126700969-14621cc0-7510-4547-9a48-1dc841145d80.png">

vs. having the word docusaurus be an anchor tag every time it's used.